API.md

```diff
@@ -3,6 +3,8 @@
 | Function | Signature |
 | -------- | --------- |
 | [`all`](#all) | `[Promise a] -> Promise [a]` |
+| [`assocWith`](#assocWith) | `String -> ({ k: v } -> a) -> { k: v } -> { k: v }` |
+| [`assocWithP`](#assocWithP) | `String -> ({ k: v } -> Promise a) -> Promise { k: v } -> Promise { k: v }` |
```

Comment:
This one is iffy, but I think the first `Promise { k: v }` should just be `{ k: v }`.  Technically, because Promises are "broken monads", either is accepted by the function safely, since `convergeP` lifts it into a Promise chain already.

But especially since this is most commonly used within a Promise chain, the final argument will almost always be a resolved value, not a Promise that resolves with a value.  So I'm thinking `{ k: v }`.